cli/src/commands/account.rs

```diff
@@ -97,12 +101,20 @@ fn aleo_literal_to_fields<N: Network>(input: &str) -> Result<Vec<Field<N>>> {
 impl Account {
     pub fn parse(self) -> Result<String> {
         match self {
-            Self::New { network, seed, vanity, discreet } => {
+            Self::New { network, seed, vanity, discreet, save_to_file } => {
                 // Ensure only the seed or the vanity string is specified.
                 if seed.is_some() && vanity.is_some() {
                     bail!("Cannot specify both the '--seed' and '--vanity' flags");
                 }
 
+                if save_to_file.is_some() && vanity.is_some() {
+                    bail!("Cannot specify both the '--save-to-file' and '--vanity' flags");
+                }
+
+                if save_to_file.is_some() && discreet {
+                    bail!("Cannot specify both the '--save-to-file' and '--discreet' flags");
+                }
```

Comment:
this would work fine, but I think [conflicts_with_all](https://docs.rs/clap/latest/clap/struct.Arg.html#method.conflicts_with_all) could be used instead together with the `clap` attribute on `save_to_file` for brevity

Reply:
After our discussion, I will keep the code as-is in this PR. Mostly because it was none-trivial to rework the code to `conflicts_with_all` combined with `--discreet`.